tests/test_model_operators.py

```diff
@@ -0,0 +1,61 @@
+import unittest
+from base64 import b64decode
+
+import bpy
+
+from mmd_tools.core.model import Model
+
+# Usage: blender --background -noaudio --python tests/test_model_operators.py -- --verbose
+
+# Due to the limitations of Windows this file can't contain characters
+# outside the cp1252 range. Because Blender parses the file with that character set.
+# http://stackoverflow.com/questions/3284827
+# https://developer.blender.org/T35176
+
```

Comment:
I tried adding that line, with the same result, seems it is ignored. Also the way the scripts are executed from the command line is different from loading addons. The process of [loading addons](https://developer.blender.org/diffusion/B/browse/master/tests/python/bl_load_addons.py) is different from executing a script from the command line. The [addon_utils](https://developer.blender.org/diffusion/B/browse/master/release/scripts/modules/addon_utils.py) python module imports found modules and calls its `register()` method. However, when executing a script from the command line, the file is loaded into memory and then the code is compiled and executed in the internal Python interpreter. So, when loading the file presumably Blender uses the system default encoding and ignores the `coding` header.